src/commands/apphosting-secrets-grantaccess.ts

```diff
@@ -9,10 +9,13 @@ import * as apphosting from "../gcp/apphosting";
 import * as secrets from "../apphosting/secrets";
 import { getBackendForAmbiguousLocation } from "../apphosting/backend";
 
-export const command = new Command("apphosting:secrets:grantaccess <secretName>")
-  .description("grant service accounts permissions to the provided secret")
+export const command = new Command("apphosting:secrets:grantaccess <secretNames>")
+  .description(
+    "grant service accounts permissions to the provided secret(s). Can pass one or more secrets, separated by a comma",
+  )
   .option("-l, --location <location>", "backend location", "-")
   .option("-b, --backend <backend>", "backend name")
+  .option("-u, --emails <emails>", "comma delmited list user or group emails")
```

Comment:
Want to update the short flag too or keep it?

```suggestion
  .option("-e, --emails <emails>", "comma delmited list user or group emails")
```